base/bucket_gocb.go

```diff
@@ -1975,6 +1978,12 @@ func (bucket CouchbaseBucketGoCB) ViewCustom(ddoc, name string, params map[strin
 		return err
 	}
 
+	// Checking and returning down here as we still need to
+	// marshal/unmarshal into vres with partial errors present.
+	if errClose != nil {
```

Comment:
It might be more readable/maintainable to switch this to `if len(viewResponse.Errors) > 0` - what do you think?

Reply:
@adamcfraser Yep, agreed.